readme.md

```diff
@@ -1576,14 +1576,14 @@ This is needed in case `process.stdin` is in [raw mode](https://nodejs.org/api/t
 
 ###### patchConsole
 
-Type: `boolean`\
+Type: `boolean | (callback: (stream: 'stdout' | 'stderr', data: string) => void) => () => void`\
 Default: `true`
 
 Patch console methods to ensure console output doesn't mix with Ink output.
 When any of `console.*` methods are called (like `console.log()`), Ink intercepts their output, clears main output, renders output from the console method and then rerenders main output again.
 That way both are visible and are not overlapping each other.
 
-This functionality is powered by [patch-console](https://github.com/vadimdemedes/patch-console), so if you need to disable Ink's interception of output but want to build something custom, you can use it.
+This functionality is powered by [patch-console](https://github.com/vadimdemedes/patch-console), so if you need to disable Ink's interception of output but want to build something custom, you can use it. You can also provide your own patch function to `patchConsole` with the same API.
```

Comment:
Maybe include your use-case as it might not be clear to the user what they would use this capability for.

Reply:
Added a basic example. Wasn't sure what to do without making it look overly complicated.